library/math.sol

```diff
@@ -8,4 +9,167 @@ library Math {
             x = mulmod(x, x, m);
         }
     }
+    
+    /// @dev unsigned constant infinity (largest number possible)
+    function Infinity() constant returns (uint inf) {
```

Comment:
After taking some time to understand how Infinity works in other languages, I think I'd agree here. Perhaps this fits better trying to make a global object in Solidity itself? Just shooting out ideas. Either way, point taken and I will update accordingly.